core/src/db/DBImpl.cpp

```diff
@@ -1024,6 +1024,7 @@ DBImpl::DeleteVectors(const std::string& collection_id, const std::string& parti
         record.lsn = 0;  // need to get from meta ?
         record.type = wal::MXLogType::Delete;
         record.collection_id = collection_id;
+        record.partition_tag = partition_tag;
         record.ids = vector_ids.data();
```

Comment:
The case "options_.wal_enable_ = true" should be modified too.
I will fix it in another PR.